packages/story-editor/src/components/canvas/singleSelectionMoveable/useResize.js

```diff
@@ -131,11 +129,17 @@ function useSingleSelectionResize({
     setTransformStyle(target, frame);
   };
 
-  const onResizeStart = ({ setOrigin, dragStart }) => {
+  const onResizeStart = ({ setOrigin, dragStart, direction }) => {
     setOrigin(['%', '%']);
     if (dragStart) {
       dragStart.set(frame.translate);
     }
+    // Both `direction[]` values for diagonals are either 1 or -1. Non-diagonal
+    // directions have 0s.
+    const newResizingMode = direction[0] !== 0 && direction[1] !== 0;
```

Comment:
```suggestion
    const newResizingMode = direction.every(n => n !== 0);
```

Reply:
Or now that I think about it again:

```suggestion
    const newResizingMode = !direction.includes(0);
```

Reply:
> Or now that I think about it again:

I'm not 100% sure about this, e.g. I don't know if `direction` includes some other indexes as well in addition to `0` and `1`. Just checking the specific indexes that we're interested in seems more safe, especially since there are only 2 of those :) Thoughts?